ext/hal/st/stm32cube/CMakeLists.txt

```diff
@@ -36,6 +36,10 @@ if(CONFIG_STM32_CORE_CM4)
   zephyr_compile_definitions( -DCORE_CM4 )
 endif()
 
+if(CONFIG_CLOCK_STM32_HSE_CLOCK)
```

Comment:
@Benichou34 
I think this change is not related and can be done as a separate PR.
Will be merged faster

Reply:
@erwango I have created a separated PR for this: [PR 19](https://github.com/zephyrproject-rtos/hal_stm32/pull/19)

Reply:
@Benichou34 , thanks. Then for current PR, I'm not sure where it will go, but it will ease things if you remove this commit.